commands/audit/jas/common_test.go

```diff
@@ -129,3 +129,24 @@ func TestSetAnalyticsMetricsDataForAnalyzerManager(t *testing.T) {
 		})
 	}
 }
+
+func TestCreateScannerTempDirectory(t *testing.T) {
+	scanner, cleanUp := InitJasTest(t)
+	defer cleanUp()
+	tempDir, err := CreateScannerTempDirectory(scanner, string(utils.Applicability))
+	assert.NoError(t, err)
+	assert.NotEmpty(t, tempDir)
+
+	// Check directory exists.
+	_, err = os.Stat(tempDir)
+	assert.NoError(t, err)
+}
+
+func TestCreateScannerTempDirectory_baseDirIsEmpty(t *testing.T) {
+	scanner, cleanUp := InitJasTest(t)
```

Comment:
```suggestion
	scanner := &JasScanner{TempDir: 'path'}
```
This is unit tests not integration. use stab